src/stories/Library/search-result-page/SearchResultPage.tsx

```diff
@@ -88,6 +90,34 @@ const SearchResult = [
   },
 ];
 
+const facetLineItems = [
```

Comment:
Consider moving this into its own separate file in the folder to avoid clutter in this component file.
And maybe while you're at it also the search result array.

Reply:
good point 👍